src/components/clusterexplorer/node.helmrelease.ts

```diff
@@ -1,21 +1,55 @@
-import * as vscode from 'vscode';
 import * as path from 'path';
-
-import { Kubectl } from '../../kubectl';
+import * as vscode from 'vscode';
+import { failed } from '../../errorable';
+import * as helmexec from '../../helm.exec';
 import { Host } from '../../host';
-import { ClusterExplorerNode, ClusterExplorerNodeImpl, ClusterExplorerHelmReleaseNode } from './node';
+import { Kubectl } from '../../kubectl';
 import { NODE_TYPES } from './explorer';
+import { ClusterExplorerHelmHistoryNode, ClusterExplorerHelmReleaseNode, ClusterExplorerNode, ClusterExplorerNodeImpl } from './node';
+import { MessageNode } from './node.message';
+import moment = require('moment');
+
+export class HelmHistoryNode extends ClusterExplorerNodeImpl implements ClusterExplorerHelmHistoryNode {
+    constructor(readonly releaseName: string, readonly revision: number, readonly updated:  string, readonly status: string, readonly description: string) {
```

Comment:
Why do you need the description here? It looks like it's never used or i'm missing something?

Reply:
Removed this, was thinking of using this but deferred to not use this